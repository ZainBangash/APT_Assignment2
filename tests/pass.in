2
./tests/TEST
place A at A1
pass
place I at B3
