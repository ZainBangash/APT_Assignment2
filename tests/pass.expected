Welcome To Scrabble!
------------------- 
Menu
----
1. New Game
2. Load Game
3. Credits
4. Quit
2

Enter the filename from which load a game
TEST
     1   2   3   4   5   6   7   8   9   10  11  12  13  14  15 
  --------------------------------------------------------------
 A |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 B |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 C |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 D |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 E |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 F |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 G |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 H |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 I |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 J |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 K |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 L |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 M |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 N |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 O |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
J-8 E-1 A-1 M-3 Y-4 L-1 Q-1 
Enter command: place A at A1

     1   2   3   4   5   6   7   8   9   10  11  12  13  14  15 
  --------------------------------------------------------------
 A | A |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 B |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 C |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 D |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 E |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 F |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 G |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 H |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 I |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 J |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 K |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 L |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 M |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 N |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 O |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
J-8 E-1 M-3 Y-4 L-1 Q-1 
Enter command: pass

I-1 E-1 R-1 O-1 O-1 R-1 T-1 
Enter command: place I at B3

     1   2   3   4   5   6   7   8   9   10  11  12  13  14  15 
  --------------------------------------------------------------
 A | A |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 B |   |   | I |   |   |   |   |   |   |   |   |   |   |   |   |
 C |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 D |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 E |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 F |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 G |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 H |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 I |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 J |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 K |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 L |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 M |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 N |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 O |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
E-1 R-1 O-1 O-1 R-1 T-1 
Enter command:
